integration-tests/jbang/src/test/resources/devmode.java

```diff
@@ -0,0 +1,27 @@
+//DEPS io.quarkus:quarkus-bom:${q.v}@pom
+//DEPS io.quarkus:quarkus-resteasy
+//JAVAC_OPTIONS -parameters
+
+import io.quarkus.runtime.Quarkus;
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+
+@Path("/hello")
+@ApplicationScoped
+public class devmode {
+
+    @GET
+    public String sayHello() {
+        return "Hello from Quarkus with jbang.dev";
+    }
+
+    @Path("/kill")
+    @GET
+    public String hello() {
+        Quarkus.asyncExit(77);
```

Comment:
@stuartwdouglas what is the proper way of shutting down quarkus when running in devmode ? I don't have the process id of quarkus; just jbang so need quarkus to shutdown.

Quarkus.asyncExit(77); works fine in non-devmode...but need solution for when running in devmode to do a full stop...System.exit() ?

Reply:
i added observer to force exit - but still something is keeping it running/hanging...